tests/interfaces/test_autograd.py

```diff
@@ -633,11 +640,15 @@ def circuit(data1, weights1, data2, weights2):
         assert res[1].shape == weights2.shape
 
         # check that the parameter shift was only performed for the
-        # differentiable input, not the data input
-        diff_variables = spy.call_args[1]["wrt"]
-        expected_diff_variables = set(data1.size + np.arange(weights1.size))
-        expected_diff_variables.update(data1.size + weights1.size + data2.size + np.arange(weights2.size))
-        assert diff_variables == expected_diff_variables
+        # differentiable elements of `weights`, not the data input
+        num_w1 = weights1.size
+        num_w2 = weights2.size
+
+        offset1 = data1.size
+        offset2 = data1.size + num_w1 + data2.size
+
+        expected = set(list(range(offset1, offset1 + num_w1)) + list(range(offset2, offset2 + num_w2)))
```

Comment:
since there's no parametrized data here, why not just write explicitly? e.g., `{6,7,8,9,10}`?

Reply:
I was about to, until I realized how long the `wrt` set was 😆 Has over 20 integers in it. Will update 👍

Reply:
done